documentation/source/refcard.rst

```diff
@@ -25,7 +25,9 @@ Reference Card
 +------------------------+-----------------------------------------------------------------+
 | Assign immediately     | ``dut.mysignal.setimmediatevalue(0xFF00)``                      |
 +------------------------+-----------------------------------------------------------------+
-| Assign metavalue       | ``dut.mysignal.value = BinaryValue("X")``                       |
+| Assign metavalue       | | ``dut.mysignal.value = Logic("X")``                           |
+|                        | | ``dut.mysignal.value = LogicArray("HLUXZ")``                  |
```

Comment:
Check me on this but IIRC LogicArray is 4-value 01XZ.
```suggestion
|                        | | ``dut.mysignal.value = LogicArray("HLUXZ")``                  |
```

Reply:
Ok, it converts from string ``"01hHlLuUwWxXzZ-"`` but itself holds only ``01XZ``.